tests/test_run_reference_vmm.py

```diff
@@ -157,7 +199,7 @@ def test_reference_vmm(kernel):
     # If we do not find the expected message, this loop will not break and the
     # test will fail when the timeout expires.
     expected_string = "Hello, world, from the rust-vmm reference VMM!"
-    expect_string(vmm_process, expected_string)
+    _ = expect_string(vmm_process, expected_string)
```

Comment:
nit: Can you delete the `_ =`? Python seems to ignore the result by default so it's not really needed. Same for lines: 171 and 185.

Reply:
Actually, in the past `pylint` has complained a lot when I didn't use output of a function that returned one, I am sure `pylint` or any of it's brethren would complain if we ignore.

Reply:
Ah, ok. I did not know that. We can then leave it as is, thanks!

Reply:
Well, I tried using `pylint` to check if it indeed complains, it didn't, so I removed as you suggested.